INSTALL.md

```diff
@@ -78,21 +81,35 @@ $ camlp5 -v
 Camlp5 version 7.03 (ocaml 4.02.3)
 ```
 
-Under Mac OS X, you may obtain ProofGeneral from http://proofgeneral.inf.ed.ac.uk/.
-It comes with installation instructions.  Your version of emacs determines
-which version of ProofGeneral you need, roughly, so some experimentation may be
-required; you may even need the current development version if your emacs is
-recent.
-
-
 ## Preparing for the installation under Linux
 
-Under Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
+uUnder Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
 
 ```bash
 $ sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp4-extra camlp5 proofgeneral proofgeneral-doc libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib
 ```
 
+Arch Linux and Manjaro Linux:
+
+    To prepare for compiling it yourself:
+
+      sudo pacman --sync --needed archlinux-keyring
+      sudo pacman-key --populate archlinux
+      sudo pacman --sync --needed ocaml camlp5 ocaml-findlib
+
+    Install ProofGeneral from its website as described above.  Or,
+    alternatively, install it from AUR (
+    https://aur.archlinux.org/packages/proofgeneral/ ) using an auxiliary
+    package manager such as "cower" ( https://aur.archlinux.org/packages/cower/
+    ) or yaourt ( https://aur.archlinux.org/packages/yaourt/ ).  That approach
+    will be especially convenient for expert Arch Linux users who find other
+    packages at AUR worth installing.
```

Comment:
I believe most Arch users are "experts" and already use cower or yaourt as a lot of packages can only be found in the AUR

Reply:
I'm an arch linux user who is not expert.

Reply:
Fair enough. I guess it cannot hurt to detailed instructions